src/refmvs.rs

```diff
@@ -34,7 +34,9 @@ use zerocopy::FromZeroes;
 #[derive(Clone, Copy, Default, PartialEq, Eq)]
 #[repr(C, packed)]
 pub struct RefMvsTemporalBlock {
-    pub mv: Mv,
+    /// This is an [`Mv`]. However, because the type [`Mv`] is `align(4)`,
+    /// we cannot include it in this packed type. Include the underlying array instead.
+    pub mv: [i16; 2],
```

Comment:
Could we use a dedicated `UnalignedMv` type instead?  One that doesn't implement `PartialEq`, `fn is_invalid`, etc?  And then have methods like `fn align(self: UnalignedMv) -> Mv` and `fn unalign(self: Mv) -> UnalignedMv` instead of the more generic `from` and `into` so it's clearer what exactly we're doing.

Reply:
Sure, I'll give that a go.